operators/pkg/controller/common/license/crud.go

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+package license
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/elastic/cloud-on-k8s/operators/pkg/controller/common"
+	"github.com/elastic/cloud-on-k8s/operators/pkg/utils/k8s"
+	pkgerrors "github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	util_errors "k8s.io/apimachinery/pkg/util/errors"
```

Comment:
According to the go doc, the use of underscores in a go package name should be avoided. I think I would go with `k8serrors` here.